handbook/process/travel-policy.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "FPA Travel policy"
+description:
+    "The policies adopted by the FPA to fund travel"
+layout: default
+---
+
+# FPA Travel policy
+
+The policies adopted by the FPA to fund travel
+
+The FreeCAD Project Association supports travel of developers and community members to in-person events to further our goal of improving FreeCAD. Travel support is subject to the following limits and guidelines:  
+
+1. The FPA sets a yearly maximum travel budget as part of its overall budgeting process, and total FPA-supported travel-related expenses cannot exceed that amount without a vote for additional appropriations. The budget is currently 15,000 USD per year.
+2. Travel support is limited to two trips per year per traveler.
+3. Standard support is capped to a maximum amount per trip (currently 600 USD per trip). Exceptions may be granted on a case-by-case basis.
+4. Support is not limited to FPA-sponsored events: any event where attendance benefits FreeCAD is eligible for support. Travel to non-FPA-sponsored events requires the traveler to explain how FreeCAD will benefit from their attendance at that event.
+5. The FPA cannot sponsor visas for international travel, but it can write an invitation letter for FPA-sponsored events. Visa expenses may be included in the travel support request. If a visa is required for attendance, airfare and lodging should be refundable. If included in the request, the FPA understands that the visa expenses are non-refundable and will be covered even in the event that the visa is denied.
+6. Travel support is provided as reimbursement based on receipts for actual expenses. If an advance is required it should be arranged by reaching out to the FPA at [fpa@freecad.org](mailto:fpa@freecad.org). The FPA does not pay a per diem rate.
+7. To request travel support, please fill out the form at [https://docs.google.com/forms/d/e/1FAIpQLSfTV6so0nFJyxnxH6-HCv0q1_393UrK2Cxjdnzl8a-a5k7d9g/viewform?usp=sf_link](https://docs.google.com/forms/d/e/1FAIpQLSfTV6so0nFJyxnxH6-HCv0q1_393UrK2Cxjdnzl8a-a5k7d9g/viewform?usp=sf_link) at least thirty days in advance of the event.
+8. Travel via train, bus, or other public transit is preferred over air travel when feasible.
+9. All travel grants are subject to a vote of the FPA members, and no support is guaranteed until that vote is completed.](https://blog.freecad.org)
```

Comment:
What is the stray link at the end of this line?

Reply:
Thanks! Solved